--- VS2010.orig/libogg.props	2017-11-29 11:47:16.000000000 +0900
+++ VS2010/libogg.props	2018-11-18 19:24:00.000000000 +0900
@@ -2,6 +2,6 @@
 <Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup Label="UserMacros">
-    <LIBOGG_VERSION>1.3.2</LIBOGG_VERSION>
+    <LIBOGG_VERSION>1.3.3</LIBOGG_VERSION>
   </PropertyGroup>
   <PropertyGroup>
     <_ProjectFileVersion>10.0.30319.1</_ProjectFileVersion>
